lib/ansible/modules/cloud/azure/azure_rm_aks.py

```diff
@@ -69,6 +69,7 @@
                 description:
                     - Number of agents (VMs) to host docker containers.
                     - Allowed values must be in the range of C(1) to C(100) (inclusive).
+                default: 1
```

Comment:
https://github.com/ansible/ansible/blob/14f5283effaa0219336e74603870bd85e3221385/lib/ansible/modules/cloud/azure/azure_rm_aks.py#L413

There is no default value, and this field is required.

Reply:
Thanks yuwzho, I will notice like this define and modify it!